### ISSUE TITLE
[Bug] VAMC Left Nav - A11y - Section headers not Tab-accessible

### ISSUE THREAD

**Issue by tlei123** [department-of-veterans-affairs/va.gov-team#8909]
## What happened?
On Staging VA Pittsburgh health care page, using keyboard nav, Left Nav menu section-headers do not receive keyboard focus.

## Specs:
- All major desktop browsers (Win Edge/IE11, Mac Safari, Win/Mac Chrome, Win/Mac Firefox)
- Unauthenticated experience

## Steps to Reproduce
1.  Go to https://staging.va.gov/pittsburgh-health-care/
1.  Tab to the last breadcrumb **VA Pittsburgh health care**.
1.  Tab once more.
1.  Observe that focus skips **SERVICES AND LOCATIONS** section-header, focusing instead on Health Services menu-option.
1.  Continue Tabbing down the menu and observe that focus skips all the section-headers.

### GIF example of SERVICES AND LOCATIONS section-header focus skipped:
![ln-a11y-bug-1-kybd-nav-200507](https://user-images.githubusercontent.com/587583/81375337-28223700-90b6-11ea-87a2-53bb3c0c1da7.gif)

## Desired behavior
All section-headers in Left Nav should be able to receive keyboard focus.

## Acceptance Criteria
\[see **Desired behavior** above]

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [x] **Attached to Epic** (what body of work is this a part of? possibly `Ongoing Maintenance`)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [x] **Labeled with `Bug`**

**Comment by brangi** [comment]
@tlei123, for this issue, I tried adding tabIndex 0 to our h2 so that the would be focusable ```  <h2
        tabIndex="0" ...>
      </h2>``` and works however, vsp has added an eslint rule that will prevent this from being used 
```
/Users/gribranrodriguez/vsa/vets-website/src/platform/site-wide/side-nav/components/NavItemRow.js
  27:9  error  `tabIndex` should only be declared on interactive elements  jsx-a11y/no-noninteractive-tabindex

✖ 1 problem (1 error, 0 warnings)

```
And I would't like to disable it. It would be nice to find an example where h2 is being tab accessible and if tabIndex is being used, it's being used differently or the tags were added before the rule was introduced.

**Comment by mmiddaugh** [comment]
@tlei123 @jenstrickland can you label this with a defect level to help us with prioritization, please?

**Comment by jenstrickland** [comment]
I don't think the section headers are meant to be tab accessible. Headings are not typically focusable, with exceptions for those that indicate a context change, such as moving from a Submit button on a form to a heading that indicates the search results. Is there documentation that indicates this should be focusable?

**Comment by mmiddaugh** [comment]
@tzelei @jenstrickland since it isn't expected that section-headers in Left Nav receive keyboard focus, can we close this issue?

**Comment by jenstrickland** [comment]
This is a QA-owned bug.

**Comment by tlei123** [comment]
Most likely a stale test case...

It's fine with me if this gets Closed, but I'd like to try some "forensics" in the interest of preventing future false-positive defects...

@jenstrickland @mmiddaugh, any idea where the relevant Expected-Result item might've come from? \[see highlighted line in screenshot below]
@pjhill , whatever you updated on 5/12, did any change(s) touch Step 1's Expected Result?

The case's one Reference #7212 has no specific mention of top-level headings receiving keyboard focus in the Tasks, and no mention of focus at all under the ACs.

https://dsvavsp.testrail.io/index.php?/cases/view/430
![Screen Shot 2020-07-08 at 2 56 08 PM](https://user-images.githubusercontent.com/587583/86975242-bfdee880-c12c-11ea-9507-43a086c07f7c.png)

Any info would be helpful in our efforts toward continuous process improvements.  Thanks much!  :)

**Comment by mmiddaugh** [comment]
This may have been my error in setting up the case, @tlei123. It may be nothing more than overenthusiastic cut and paste and if so, I apologize for the inconvenience and confusion.

**Comment by tlei123** [comment]
Thx, @mmiddaugh !  No sweat.  It'll prob'ly jump out at you next time you see the same copy-source.

**Comment by pjhill** [comment]
FWIW -- My update on 5/12 was just to change the `Reference` field from a full blown URL to just the number of the ticket being referenced in GitHub, so the TestRail->GH integration would fire off a bit better.

![image](https://user-images.githubusercontent.com/52456633/87311632-7e17be80-c4ed-11ea-9000-fbc20ec3c933.png)
